index.js

```diff
@@ -573,10 +573,18 @@ class BitbucketScm extends Scm {
      * @param  {String}   config.token        The token used to authenticate to the SCM
      * @param  {String}   config.url          Target Url of this commit status
      * @param  {String}   [config.jobName]    Optional name of the job that finished
+     * @param  {Number}   [config.pipelineId] Pipeline ID
      * @return {Promise}
      */
     _updateCommitStatus(config) {
+        let context = `Screwdriver/${config.pipelineId}`;
```

Comment:
It's set to `optional` so that it doesn't break during the time [data-schema](https://github.com/screwdriver-cd/data-schema/pull/144) was merged, but [models](https://github.com/screwdriver-cd/models/pull/172) was not merged. It's for backward compatibility. 

After API pulls in the new models, this field will always be passed in: https://github.com/screwdriver-cd/models/blob/master/lib/build.js#L54